test-framework/sudo-compliance-tests/src/snapshots/sudoers/user_list/user_alias_works.snap

```diff
@@ -1,5 +1,5 @@
 ---
 source: sudo-compliance-tests/src/sudoers/user_list.rs
-expression: output.stderr()
+expression: stderr
 ---
-ghost is not in the sudoers file.  This incident will be reported.
```

Comment:
Are you telling me that incidents won't be reported?? You'll regret this!

Reply:
Sudo will not do [Santa's work](https://imgs.xkcd.com/comics/incident.png) for him anymore.

Reply:
Nice, I'll finally get my RISC-V SoC this year